lua/lspconfig/server_configurations/grammarly.lua

```diff
@@ -0,0 +1,33 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    filetypes = { 'markdown' },
+    root_dir = util.find_git_ancestor,
+  },
+  docs = {
+    description = [[
+Grammarly in an LSP, using the Grammarly API.
+
+Requires some manual installation:
+
+1. Clone the Grammarly repo, the upstream is [here](https://github.com/znck/grammarly), it's designed to work with a custom client though, so it requires some modifications. I'd recommend using [this branch of my fork](https://github.com/mtoohey31/grammarly/tree/generic-client-compat) might work for you.
```

Comment:
The language here is a bit colloquial, no first/second person.

Reply:
Fixed by 5669ee8dd4982003b65fa92795e2bb18e909b302.